### ISSUE TITLE
Theme won't work 4.26.2

### ISSUE THREAD

**Issue by elamhut** [hugoattal/DarkerNodes#142]
![UE4Editor_oQDR4PrS6c](https://user-images.githubusercontent.com/36922414/116008047-19939f00-a5e9-11eb-8d52-9fa40cbf3fc3.jpg)

The plugin won't work properly on my Unreal. Color work fine but no Icon works and the top window bar is still the default style. I've tried clicking everything. Also the "Reload Default Style" button won't work no matter what I do (yes, I know I have to uncheck the Master Activate, even then it's not clickable).

**Comment by hugoattal** [comment]
@elamhut Hum, that's pretty weird... Is it your first install? Or did you update the engine or the plugin recently?
Can you try to uninstall/reinstall it from the launcher? (Can you make sure you have the 3.2 version also?)

> Also the "Reload Default Style" button won't work no matter what I do
Here are the steps:
- Disable "Master Activate"
- Click on "Reload Default Style"
- Restart the engine

It didn't do anything?

---

This is a really weird issue, would you mind a small discord screen sharing by any change?

**Comment by hugoattal** [comment]
I just created a new project on 4.26.2 just to make sure it's not a common thing on the new update, I didn't have any issue...

I suggest disabling/uninstalling Darker Nodes and reinstalling again from the marketplace :/ ... I'm very sorry about this...

**Comment by elamhut** [comment]
Using Windows 10 currently.
Trying to reload default style: 

https://youtu.be/mtd1GxxgaQQ

Uninstalling / Reinstalling:
https://i.imgur.com/efz368d.png

https://i.imgur.com/zEyWIla.png

https://i.imgur.com/sFIuTdo.png

https://i.imgur.com/QwfDK5B.png

https://i.imgur.com/G6Ge9Zd.png

Still not working:
https://i.imgur.com/G6Ge9Zd.png

I don't know if this has anything to do with it but I use dual 4k monitors with Enabled High DPI Settings.

**Comment by hugoattal** [comment]
> https://youtu.be/mtd1GxxgaQQ

I think you're missing the last step:
- Disable "Master Activate"
- Click on "Reload Default Style"
- **Restart the Unreal Engine** (leaving the "master activate" disabled)

I guess I should simplify this process 🤔 ...

---

> I don't know if this has anything to do with it but I use dual 4k monitors with Enabled High DPI Settings.

I can't see any reason why it wouldn't work with such settings... Still, maybe try to uncheck "Enable High DPI" just to see if that fixes anything? I suggest you try those solutions for another graphical issue:
https://github.com/hugoattal/DarkerNodes/wiki/FAQ#the-unreal-editor-menus-are-flickering

---

When you uninstalled Darker Nodes, could you make sure that this folder does not exist anymore? There has been some weird rare case of Epic not uninstalling everything and files from some older version stayed in.
`C:\Program Files\Epic Games\UE_4.26\Engine\Plugins\Marketplace\DarkerNodes`

---

I'm really sorry you encounter such a problem, I'm just trying to figure out where does this come from, as it's the 1st time I see such bug... If none of these works, would you mind a small screen sharing for me to investigate a bit further? I really don't want to waste your time though.

**Comment by elamhut** [comment]
Ok, so I did a fresh Unreal 4.26.2 install, deleted EVERYTHING, even from registry, moved its install folder to a new drive. I have some good news and bad news...

The good news is that the plugin on a fresh project works fine but on my main project it still doesn't work.

https://i.imgur.com/fdhFPnu.jpg

No worries, I really want to have this plugin working fine :/ If I can manage to make it work I'll get the Nodes plugin as well, but I need this one working first :/

**Comment by hugoattal** [comment]
Oh, so it's specific to your project? Did you create your project on 4.26 or did you update it from a previous engine version?

Would you mind sharing a zip of a copy of your project with everything deleted? I'm looking for a minimal example so that I can reproduce it on my end and fix it.

Again, I'm really sorry for all the troubles :/...

**Comment by elamhut** [comment]
Upgraded from 4.25

Plugin wise we have:
Source Control Plugin (Plastic)
FMod Plugin
Substance Plugin
Linter Plugin
Node Graph Assistant Plugin
Modeling Tools Editor Mode Plugin
We have disabled Lightmap engine-wide for the project

**Comment by hugoattal** [comment]
(On a side note, this can be fixed with "Fix Missing Elements" in the settings)
![image](https://user-images.githubusercontent.com/4563971/116123933-e7a33b00-a6c3-11eb-9d09-e65594c8dac1.png)

**Comment by elamhut** [comment]
> Would you mind sharing a zip of a copy of your project with everything deleted? I'm looking for a minimal example so that I can reproduce it on my end and fix it.

What do you mean everything deleted? the content folder?

**Comment by elamhut** [comment]
Ok so I went disabling our plugins one by one...

When I disabled FMod:
https://i.imgur.com/FnBB18e.jpg

**Comment by hugoattal** [comment]
Okay, so FMod might update the style of the editor after Darker Nodes! I'll download it and see how to fix it. I'll come back to you as soon as I find a solution!

Thank you very much for your help :D !

**Comment by elamhut** [comment]
No problem! Just confirmed that renabling FMod breaks Darker Nodes. Our FMod is the latest version for 4.26

**Comment by hugoattal** [comment]
@elamhut Ok, so I dig down this issue and understood a bit better.

TLDR, here is a temporary fix:
- Find the Darker Nodes directory
(usually something like `C:\Program Files\Epic Games\UE_4.26\Engine\Plugins\Marketplace\DarkerNodes`)
- Edit the `DarkerNodes.uplugin` files
- Replace `"LoadingPhase": "PostDefault"` with `"LoadingPhase": "PostEngineInit"`
- Restart the engine

Why is there a bug? It appears that FMODStudio update the style of the engine by recreating a new one from the engine and then applying it. Thing is it's erasing other modication previously done:
https://github.com/fmod/ue4integration/blob/2.01-UE4.26/FMODStudio/Source/FMODStudioEditor/Private/FMODStudioStyle.cpp#L22

And they do that pretty late in the loading of the engine, in "PostEngineInit", so they erase a bunch of modification Darker Nodes does before. I'll create an issue on their side to see if they can update this a bit.
https://github.com/fmod/ue4integration/blob/2.01-UE4.26/FMODStudio/Source/FMODStudioEditor/Private/FMODStudioEditorModule.cpp#L288

**Comment by hugoattal** [comment]
I created an issue on their side: https://github.com/fmod/ue4integration/issues/62

**Comment by elamhut** [comment]
https://i.imgur.com/I5CDZv0.jpg

It wooooorks! Nice :D

Also purchased Electronic Nodes, great asset, really amazing I love it.

Just sometimes the nodes have a really really small misalignment that corrodes my inner being and no ammount of Pressing Q to align the nodes are able to fix:
https://i.imgur.com/nZ0HzLn.png

Otherwise thanks a ton for the help :)
